profile.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Profile page</title>
+    <link rel="stylesheet" href="/dist/css/styles.css">
+</head>
+
+<body>
+    <main>
```

Comment:
Any reason why the navbar is inside the main element? I think the navbar is good enough to be on its own. It is semantically incorrect.

Reply:
No other reason then summer rustiness. Fixed.